docker-compose/README.md

```diff
@@ -125,8 +125,26 @@ This example starts the following containers
 
 Artifactory uses the PostgreSQL database running in another container.
 
+#### Upgrade Artifactory Pro with PostgreSQL only
+```bash
+### Linux
+1. $ sudo docker-compose -f artifactory-pro-postgresql.yml stop
+2. Edit the artifactory-pro-postgresql.yml file and change the artifactroy image version to the version you would like to upgrade to
```

Comment:
To save downtime, the `artifactory-pro-postgresql.yml` can be edited first and only then run `stop` -> `up`.
I suggest you change the order. Test this.